Tests/SwiftFrameTests/Utility/ConfigDataMock.swift

```diff
@@ -0,0 +1,35 @@
+import AppKit
+import Foundation
+@testable import SwiftFrameCore
+
+extension ConfigData {
+
+    static var goodData: Self {
+        ConfigData(
+            clearDirectories: true,
+            outputWholeImage: true,
+            textGroups: nil,
+            stringsPath: FileURL(path: "testing/strings/"),
+            maxFontSize: 200,
+            outputPaths: [FileURL(path: "testing/output/")],
+            fontSource: .nsFont(.systemFont(ofSize: 20)),
+            textColorSource: try! ColorSource(hexString: "#ff00ff"),
+            outputFormat: .png,
+            deviceData: [DeviceData.goodData])
```

Comment:
Might be able to omit `DeviceData` here and just use `.goodData`. Similar in other places.